packages/addons/service/nextpvr/source/addon.py

```diff
@@ -75,6 +75,8 @@ def updateNextPVR(self):
             xbmc.log('Running: %s' % command, xbmc.LOGDEBUG)
             os.system(command)
             os.remove(archive)
+            command = 'find {0}/DeviceHost -name DeviceHostLinux -exec chmod 755 {{}} \;'.format(dest_folder)
```

Comment:
Shouldn't this be in the nextpvr-downloader, and be a fully qualified path not variable driven?

Reply:
There are two ways of downloading, the default in nextpvr-downloader does download a specific version based, this second manual download in python gets the current Linux build, if a user might want to update LE11 or download a new version before it is packaged for LE 12/13.  The generic build has no version attributes.